deps/spidershim/src/v8trycatch.cc

```diff
@@ -41,7 +41,7 @@ struct TryCatch::Impl {
     // TODO: Propagate exceptions to the caller
     //       https://github.com/mozilla/spidernode/issues/51
   }
-  Isolate* Isolate() const { return isolate_; }
+  Isolate* Iso() const { return isolate_; }
```

Comment:
In this case, the `isolate_` member is already declared as `class Isolate*`, so I think everything would be fine here with my original suggestion.  I did that in 9176b11.  If that breaks the build for you, please file an issue.  Thanks!